cloud/aws/services/ec2/instances.go

```diff
@@ -25,6 +25,24 @@ import (
 	"sigs.k8s.io/cluster-api-provider-aws/cloud/aws/providerconfig/v1alpha1"
 )
 
+const (
+	controlPlaneUserData = `#!/usr/bin/env bash
```

Comment:
this isn't going to be a const for very long, i'm embedding certs and generating this init script.

this could be a const as a format, but i wasn't quite done working with it